benchmarks/apps/cp2k/cp2k.py

```diff
@@ -3,13 +3,9 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-import os.path as path
```

Comment:
`path` is used below.

Reply:
Thanks for spotting! I looked for any use of `os` in the apps, but didn't notice it's imported as just `path`